src/components/Root/FFetch.js

```diff
@@ -77,10 +77,27 @@ const OKAPI_FETCH_OPTIONS = {
 };
 
 export class FFetch {
-  constructor({ logger, store, rtrConfig }) {
+  constructor({ logger, store }) {
     this.logger = logger;
     this.store = store;
-    this.rtrConfig = rtrConfig;
+  }
+
+  /**
+   * registers a listener for the RTR_FORCE_REFRESH_EVENT
+   */
+  registerEventListener = () => {
+    this.globalEventCallback = () => {
+      this.logger.log('rtr', 'forcing rotation due to RTR_FORCE_REFRESH_EVENT');
+      rtr(this.nativeFetch, this.logger, this.rotateCallback, this.store.getState().okapi);
+    };
+    window.addEventListener(RTR_FORCE_REFRESH_EVENT, this.globalEventCallback);
+  }
+
+  /**
+   * unregister the listener for the RTR_FORCE_REFRESH_EVENT
+   */
+  unregisterEventListener = () => {
+    window.removeEventListener(RTR_FORCE_REFRESH_EVENT, this.globalEventCallback);
   }
```

Comment:
All our other event handlers are consolidated within SessionEventContainer. Can we move this there too? If not why not? 

Functionally it should not matter either way, but it would be nice to keep similar things in the same place.

Reply:
Unfortunately, I don't think this can be consolidated; this handler specifically needs access to those callbacks and fields within `FFetch`, which `SessionEventContainer` has no way to access